crates/dips/src/lib.rs

```diff
@@ -479,7 +483,7 @@ mod test {
                 payee: payee_addr,
                 service: Address(FixedBytes::ZERO),
                 metadata: metadata.eip712_hash_struct().to_owned().into(),
-                cancellled_by: signer.address(),
+                cancelledBy: signer.address(),
```

Comment:
This is still British English spelling, "canceled" in US spelling fwiw fyi,

Reply:
Good point, I never remember which is which... I see other references to "cancelled" in the repo though (it seems tokio uses cancelled in cancellation tokens https://docs.rs/tokio-util/latest/tokio_util/sync/struct.CancellationToken.html which we use here), so I'd rather leave this as it is for consistency

Reply:
TIL on the CancellationToken! As long as we're being consistent then lol